inc/notificationtargetcommonitilobject.class.php

```diff
@@ -631,8 +631,16 @@ function getDatasForTemplate($event, $options=array()) {
              && isset($options['items'])) {
             $entity = new Entity();
             if ($entity->getFromDB($options['entities_id'])) {
-               $this->datas["##$objettype.entity##"]      = $entity->getField('completename');
-               $this->datas["##$objettype.shortentity##"] = $entity->getField('name');
+               $this->datas["##$objettype.entity##"]          = $entity->getField('completename');
```

Comment:
I know this code already exists but... Would it be possible to `s/datas/data/`? ;)